.changeset/blue-rings-live.md

```diff
@@ -0,0 +1,27 @@
+---
+"wrangler": patch
+---
+
+feat: implement `rules` config field
+
+This implements the top level `rules` configuration field. It lets you specify transport rules for non-js modules. For example, you can specify `*.md` files to be included as a text file with -
+
+```
+[[rules]]
+{type = "Text", globs = ["**/*.md"]}
+```
+
+We also include a default ruleset -
+
+```
+  { type: "Text", globs: ["**/*.txt", "**/*.html", "**/*.pem"] },
```

Comment:
`.pem` seems like a weird default to have, could this be left out?

Reply:
yeah for sure

Reply:
`.pem` appears to have been supported prior to this PR... 

https://github.com/cloudflare/wrangler2/pull/530/files/22aa1a4ba797f6e5b5793505cc61169ccc1e391c#diff-17e01c57aa611bb9e0b392a15fd63b5d18602e3a6c9a97c4a34e891bbfdcb7f3L47